contrib/vmap_extractor/vmapextract/model.cpp

```diff
@@ -24,7 +24,7 @@
 #include <algorithm>
 #include <cstdio>
 
-Model::Model(std::string& filename) : filename(filename), vertices(0), indices(0)
+Model::Model(std::string& filename) : vertices(0), indices(0), filename(filename)
```

Comment:
Compiler warnings again. Class members are initialized in the order they are declared, no matter what order you're using in the definition. Some compilers, like GCC and clang, are warning you that the order you've written the members in, is not what you actually get.